.gitlab-ci.yml

```diff
@@ -1,4 +1,9 @@
 ---
+stages:
+  - .pre
+  - test
+  - build
+  - .trigger
```

Comment:
Does the prefix `"."` mean anything?

Reply:
I have a job `trigger` and a stage `.trigger` .
Just indicating this is a stage.

Reply:
nit:
maybe a renaming would be appropriate in the future?

```diff
-  - .trigger
+  - stage-trigger
```